### ISSUE TITLE
Add participant difficulty data back

### ISSUE THREAD

**Issue by mtakane** [openpracticelibrary/openpracticelibrary#315]
Problem: Upon the move to do facilitator difficulty as meta data, it seems to have lost a similar means to define participant meta data.

Solution: create a similar field or capability as what is done for the facilitator difficulty

**Comment by springdo** [comment]
@mtakane  - I am not sure I agree with this. I would imagine the how to's difficulty would only refer to the facilitator or both if anything.

**Comment by rdebeasi** [comment]
I agree with @springdo on this. I'd prefer to keep things concise by having only one "difficulty" field. I'd argue that in the majority of cases, participant and facilitator line up reasonably well. For example, Event Storming is fairly difficult for participants and facilitators alike.

For practices that are easy for participants and difficult for facilitators (or vice-versa), maybe we can call those out in the article text.

Thought?

**Comment by mtakane** [comment]
@springdo @rdebeasi While I typically see only a single difficulty field on things. One feedback that was a pretty enthusiastic one was how it helped differentiated this Library over others by sharing how the difficulty of practices varies for the facilitator and the participants. I don't want to rule it out until we get some more confirmation from other users on if this is a good or bad idea.

I can see myself as a user that I don't really care about facilitator difficulty (cause it is me and I'll spend the time to learn it), but I want something that is pretty easy for participants. How can I find that type of content, buried in article text?....

**Comment by rdebeasi** [comment]
Can we find an example of a practice that has different facilitator/participant difficulty? Having a real example might help us find a good solution.

**Comment by rdebeasi** [comment]
> If we do end up having on a single difficulty field, it should no longer say "facilitator difficulty" then.

The difficulty field now just says "difficulty".

I went through our practices, and participant difficulty generally tracks pretty closely with facilitator difficulty. For example, "definition of done" is easy to participate in and easy to facilitate. "Event storming" is hard to participate in and hard to facilitate.

In general, my experience has been that facilitating a practice is generally more difficult than participating, regardless of the practice. So if we were going to get really granular, we could say that an "easy" practice is super-easy to participate in, and a "medium" practice is fairly easy to participate in.

Long story short, if you want something that's easy for participants, a practice marked with the difficulty "easy" or "medium" will get you what you're looking for. I agree that we could do a better job of helping facilitators choose the right practice for a given audience, but I think a "facilitation" mindset would do this better than an extra piece of metadata.

Would it be okay if I closed this issue?

**Comment by rdebeasi** [comment]
This seems to be resolved and I haven't heard back on this for a few weeks, so I'm going to close this issue. If I've done this in error, just let me know and I'll reopen. Thanks! :smile:
